HD-Map/hdmap_shacl.ttl

```diff
@@ -0,0 +1,401 @@
+@prefix hdmap: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/hdmap/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+hdmap:hdmapShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:FormatShape ;
+            sh:order 5 ;
+            sh:path hdmap:Format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:ContentShape ;
+            sh:order 2 ;
+            sh:path hdmap:Content ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:GeneralShape ;
+            sh:order 3 ;
+            sh:path hdmap:General ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:QuantityShape ;
+            sh:order 0 ;
+            sh:path hdmap:Quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:DataSourceShape ;
+            sh:order 6 ;
+            sh:path hdmap:DataSource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:QualityShape ;
+            sh:order 1 ;
+            sh:path hdmap:Quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node hdmap:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path hdmap:Georeference ] ;
+    sh:targetClass hdmap:hdmap .
```

Comment:
Dont forget to change the targetClass to HdMap when ontology is changed.

Reply:
fixed